src/playbacks/html5_video/html5_video.js

```diff
@@ -95,6 +95,7 @@ export default class HTML5Video extends Playback {
     super(...args)
     this._destroyed = false
     this._loadStarted = false
+    this._bufferingState = false
```

Comment:
I agree with you, I only maintained the old variable name, I changed this in latest commits :)